docs/workflow/building/libraries/README.md

```diff
@@ -238,11 +238,11 @@ To build a library's package, simply invoke `dotnet pack` on the src project aft
 
 ```cmd
 build libs
-dotnet pack src\libraries\System.Text.Json\src\
+dotnet.cmd/.sh pack src\libraries\System.Text.Json\src\
```

Comment:
`dotnet.sh pack src\libraries\System.Text.Json\src\` does not work on unix:
```sh
# wrong
$ dotnet.sh pack src\libraries\System.Text.Json\src\
zsh: command not found: dotnet.sh

# wrong
$ ./dotnet.sh pack src\libraries\System.Text.Json\src\
MSBuild version 17.6.0-preview-23108-10+51df47643 for .NET
MSBUILD : error MSB1009: Project file does not exist.
Switch: srclibrariesSystem.Text.Jsonsrc

$ ./dotnet.sh pack src/libraries/System.Text.Json/src
# works!
```
since this block is tagged ` ```cmd` (windows command prompt), i'd just drop `/.sh` part.

```suggestion
dotnet.cmd pack src\libraries\System.Text.Json\src\
```